great-idea/services.html

```diff
@@ -1,107 +1,153 @@
+            rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur
+            feugiat justo quis aliquet. Curabitur cursus et justo eu consequat.
+            Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend.
+            Mauris euismod facilisis iaculis.
+       </p> 
+       <button>Learn More</butto>
+        </article>
+
+        <article>
+          <h2>Digital Marketing</h2> 
+          <p class= learnText>
+            Nulla suscipit interdum diam at maximus. Proin vitae
+            tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+            consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+            consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.
+          </p> 
+          <button>Learn More</butto>
+        </article>
+
+        <article>
+          <h2>Web Development</h2> 
+          <p class= learnText>
+            Nulla suscipit interdum diam at maximus. Proin vitae
+            tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+            consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+            consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.
+          </p> 
+          <button>Learn More</butto>
+        </article>
+
+        <article>
+          <h2>iOS Development</h2> 
+          <p class= learnText>
+            Nulla suscipit interdum diam at maximus. Proin vitae
+            tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+            consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+            consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.
+          </p> 
+          <button>Learn More</butto>
+        </article>
+
+        <article>
+          <h2>Android Development</h2> 
+          <p class= learnText>
+            Nulla suscipit interdum diam at maximus. Proin
+            vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+            consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+            consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.
+          </p> 
+          <button>Learn More</butto>
+        </article>
+      </div>
+<hr>
+     
+        <h2>Some Facts About Our Services</h2> 
+        <p class= learnText>
+          Nulla suscipit interdum diam at maximus.
+          Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo
+          eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.
+        </p> 
+        <div class=servicesEnd>
+      <p>
```

Comment:
for bullets generally you wrap them in `<ul>` tags.  It stands for unordered list